### PR TITLE
Refactor FXIOS-4831 [v106] Update L10n intro test

### PR DESCRIPTION
**Pull request by clarmso**
The L10n intro test is failing because the design has been updated (see #11733). In particular the button to swipe to the next screen is changed. The test has been updated to reflect the update.

### CODE REVIEW

File: L10nSnapshotTests/L10nSuite1SnapshotTests.swift
Comment (isabelrios) [inline L10nSnapshotTests/L10nSuite1SnapshotTests.swift:44]: Whe I opened the issue there were only two onboarding cards, not sure we need this part (and if it is working) unless there is a new card introduced that I'm not aware of 🙈

File: L10nSnapshotTests/L10nSuite1SnapshotTests.swift
Comment (isabelrios) [inline L10nSnapshotTests/L10nSuite1SnapshotTests.swift:45]: Actually I have just tested that the test works but with this click we go to the Homescreen.
So, I suggest, either remove this part (from line 44), or rename the Snapshot in L47 as "Homescreen first visit"

File: L10nSnapshotTests/L10nSuite1SnapshotTests.swift
Comment (isabelrios) [inline L10nSnapshotTests/L10nSuite1SnapshotTests.swift:38]: Oh looks like Swiftlint is complaining about this line:
https://github.com/mozilla-mobile/firefox-ios/pull/11794/checks?check_run_id=8226000705
```
🛑 1 out of 1 tests have failed!
/Users/vagrant/git/L10nSnapshotTests/L10nSuite1SnapshotTests.swift
Lines should not have trailing whitespace.

warning:
Line:38 

```

**Review by isabelrios — APPROVED** [review]
LGTM, once the Swiftlint rule is passing, we can land this. Thanks!

File: L10nSnapshotTests/L10nSuite1SnapshotTests.swift
Comment (clarmso) [reply L10nSnapshotTests/L10nSuite1SnapshotTests.swift]: ohno
Line 38 (a blank line) begins with a tab.

**Pull request merged**
